modules/home-manager/vesktop.nix

```
@@ -0,0 +1,29 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  cfg = config.catppuccin.vesktop;
+  themeName = "catppuccin-${cfg.flavor}-${cfg.accent}.theme";
+in
+
+{
+  options.catppuccin.vesktop = catppuccinLib.mkCatppuccinOption {
+    name = "vesktop";
+    accentSupport = true;
+  };
+
+  config = lib.mkIf cfg.enable {
+    programs.vesktop.vencord = {
+      settings.enabledThemes = [ "${themeName}.css" ];
+      themes."${themeName}" = ''
+        /**
+         * @name Catppuccin ${catppuccinLib.mkFlavorName cfg.flavor} (${catppuccinLib.mkUpper cfg.accent})
```

We removed `catppuccinLib.toUpper` since we upstreamed it to nixpkgs see https://noogle.dev/f/lib/toSentenceCase
```suggestion
         * @name Catppuccin ${catppuccinLib.mkFlavorName cfg.flavor} (${lib.toSentenceCase cfg.accent})
```